### PR TITLE
Item details

### PR DESCRIPTION
**Pull request by ghost**
details

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "codelyzer": "^6.0.0",
     "jasmine-core": "~3.6.0",
     "jasmine-spec-reporter": "~5.0.0",
-    "karma": "~5.1.0",
+    "karma": "^6.3.2",
```
Comment (TalShperling) [inline package.json:52]: Why did you need to upgrade this?

File: src/entities/item.model.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 export interface Item {
     id: string;
     name: string;
+    amount?: number;
```
Comment (TalShperling) [inline src/entities/item.model.ts:4]: Are we willing to keep this?

**Comment by Ozzorany** [comment]
@LiorVais 
Just fix conflicts, and use your item details modal in items management too, just add it there

- [x] conflicts

- [x] items management- use your item info modal

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json:52]: Just because npm said its outdated and has vulnerabilities

File: src/entities/item.model.ts (hunk 2 above)
Comment (ghost) [reply src/entities/item.model.ts:4]: idk
optional for now
